grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -451,17 +451,13 @@ class IndicatorDataService {
             } else {
                 results[yearLabel] = it[0]
             }
-
-            // Check if the list with labels contains yearLabel
-            if (!listLabel.contains(yearLabel)) {
-                listLabel << yearLabel
-            }
         }
+        results = results.sort()
```

Comment:
out of curiosity - what is it sorted by? I ask, because I don't see what type is hidden under the `results`

Reply:
`results` is of type `LinkedHashMap`, apparently by default it sorts its values by keys